libraries/AR_Motors/AP_MotorsUGV.h

```diff
@@ -176,6 +176,9 @@ class AP_MotorsUGV {
     // slew limit throttle for one iteration
     void slew_limit_throttle(float dt);
 
+    // clear all limit flag
+    void clear_limits() { limit.steer_left = limit.steer_right = limit.throttle_lower = limit.throttle_upper = false; }
```

Comment:
I'm not really a fan of these little functions if there only used in one place, just makes it harder to see what is going on IMHO.

Reply:
your wish is my command.  removed.